image-layout.md

```diff
@@ -1,13 +1,13 @@
 ## Open Container Initiative Image Layout Specification
 
-The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
+The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) reference tags (tags).
```

Comment:
On Mon, Aug 29, 2016 at 02:47:15PM -0700, Brandon Philips wrote:
> How is calling this a "tag" stopping you from having multiple
> image-names?

Ah.  If that's the intention, then I'd suggest the refs/ → tags/
rename [1] if you need to do anything at all.  Introducing
“references” and “tags” as two ways of talking about the same thing
(location-addressed Descriptors) seems like an unnecessary source of
confusion.

And it means we keep the “allowing for lots of different refs” case
you call out in [2] but don't address in the bd370ef2 diff.

I don't see much point in churning the name, but I'm ok with name
churn as long as image-layout keeps the position that refs (or tags,
or whatever you want to call the mutable, location-addressable
entries) are have no semantics outside of what's carried by the
Descriptor.

[1]: https://github.com/opencontainers/image-spec/pull/219/files#r76690262
[2]: https://github.com/opencontainers/image-spec/pull/219#issue-173818111